src/components/ol/Map.vue

```diff
@@ -109,13 +110,14 @@ export default {
       olproj4(proj4);
     }
 
-    // Projection for Map, default is Web Mercator
+    // Projection for map, default is Web Mercator
+    let projection;
     if (!this.projection) {
```

Comment:
spontaneously I would turn around the if/else - e.g. `if (this.projection){ ... } else {}` 
I find  this more  readable. But that is pure eye-candy  ...